generators/common/templates/README.md.ejs

```diff
@@ -293,7 +302,7 @@ To stop it and remove the container, run:
 You can also fully dockerize your application and all the services that it depends on.
 To achieve this, first build a docker image of your app by running:
 
-    <% if (buildTool === 'maven') { %>./mvnw package -Pprod verify jib:dockerBuild<% } %><% if (buildTool === 'gradle') { %>./gradlew bootWar -Pprod jibDockerBuild<% } %>
+    <% if (buildTool === 'maven') { %>./mvnw package -Pprod verify jib:dockerBuild<% } %><% if (buildTool === 'gradle') { %>./gradlew bootJar -Pprod jibDockerBuild<% } %>
```

Comment:
FYI: `bootJar` may not be necessary. Jib doesn't look into a packaged JAR, so normally it's enough to do `./gradlew jibDockerBuild`, unless you wired up extra tasks with `bootJar`.

Reply:
I wished our build could be as fast as just compiling the sources but we have a complex builds with resource copying and filtering and in some case even a front-end nodejs build to run. However I wish we could create a special mode to speed up everything and have a fast dev workflow with jib + scaffold. Anyway thanks a lot for your help !